scipy/stats/tests/test_stats.py

```diff
@@ -6010,26 +6010,35 @@ def test_describe_empty(self):
         assert_raises(ValueError, stats.describe, [])
 
 
-def test_normalitytests():
+@pytest.mark.skip_xp_backends(cpu_only=True,
+                              reasons=['Uses NumPy for pvalue'])
+@pytest.mark.usefixtures("skip_xp_backends")
+@array_api_compatible
+def test_normalitytests(xp):
     assert_raises(ValueError, stats.skewtest, 4.)
     assert_raises(ValueError, stats.kurtosistest, 4.)
     assert_raises(ValueError, stats.normaltest, 4.)
 
     # numbers verified with R: dagoTest in package fBasics
-    st_normal, st_skew, st_kurt = (3.92371918, 1.98078826, -0.01403734)
-    pv_normal, pv_skew, pv_kurt = (0.14059673, 0.04761502, 0.98880019)
+    st_normal, st_skew, st_kurt = (3.92371918, xp.asarray(1.98078826), -0.01403734)
+    pv_normal, pv_skew, pv_kurt = (0.14059673, xp.asarray(0.04761502), 0.98880019)
     pv_skew_less, pv_kurt_less = 1 - pv_skew / 2, pv_kurt / 2
     pv_skew_greater, pv_kurt_greater = pv_skew / 2, 1 - pv_kurt / 2
-    x = np.array((-2, -1, 0, 1, 2, 3)*4)**2
+    x = np.array((-2, -1, 0, 1, 2, 3.)*4)**2
+    x_xp = xp.asarray((-2, -1, 0, 1, 2, 3.)*4)**2
```

Comment:
Slightly messy having this twice but once all are converted we can get rid of the numpy one. Needed as the default numpy dtype is float64 so when we convert to torch we get float64 when we probably wanted the default float32